archetype/test/integration/src/test/java/com/liferay/faces/bridge/ext/archetype/test/integration/ArchetypePortletsTester.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2000-2017 Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+package com.liferay.faces.bridge.ext.archetype.test.integration;
+
+import java.util.Locale;
+
+import org.junit.Test;
+
+import com.liferay.faces.test.selenium.browser.BrowserDriver;
+import com.liferay.faces.test.selenium.browser.BrowserDriverManagingTesterBase;
+import com.liferay.faces.test.selenium.browser.TestUtil;
+import com.liferay.faces.test.selenium.browser.WaitingAsserter;
+
+
+/**
+ * @author  Kyle Stiemann
+ */
+public class ArchetypePortletsTester extends BrowserDriverManagingTesterBase {
+
+	// Private Constants
+	private String[] ARCHETYPE_LIBRARIES = {
+			"Alloy", "ICEfaces", "jsf", "PrimeFaces", "RichFaces", "BootsFaces", "ButterFaces"
```

Comment:
@stiemannkj1 Might be good to remove BootsFaces and ButterFaces in the constants since they are only for JSF 2.2 and the bridge-ext-4.x branch is for JSF 2.1 at this time.

Reply:
@ngriffin7a, fixed: https://github.com/liferay/liferay-faces-bridge-ext/pull/143